README.md

```diff
@@ -59,19 +59,19 @@
 
 1) Presentation
 
-   Open Source Firmware Conference 2020 - [openspdm](https://cfp.osfc.io/osfc2020/talk/ECQ88N/)
+   Open Source Firmware Conference 2020 - [libspdm](https://cfp.osfc.io/osfc2020/talk/ECQ88N/)
 
-   Free and Open Source Developers European Meeting 2021 - [openspdm](https://fosdem.org/2021/schedule/event/firmware_uoifaaffsdc/)

```

Comment:
This can stay openspdm since that was the name at the time.

Reply:
Yes. I have changed. Thanks.